.github/ISSUE_TEMPLATE/tech_debt.md

```diff
@@ -0,0 +1,29 @@
+### Description
+A clear and concise description of what the tech debt is and the reason of being created
+
+### Impact
+Description of the current or future impact of this tech debt.
+
+### Risk
+Description of risks associated with leaving this debt unresolved.
+
+### Proposed Solution(s)
+##### Solution 1.
+| How | _______ |
+| :---: |  :---: |
+| Pros :green_heart: | _______ |
+| Cons :broken_heart: |  _______  |
+| Effort | SMALL / MEDIUM / LARGE |
+
+##### Solution 2.
+| How | _______ |
+| :---: |  :---: |
+| Pros :green_heart: | _______ |
+| Cons :broken_heart: |  _______  |
+| Effort | SMALL / MEDIUM / LARGE |
+
+
+### Acceptance Criteria
+| Given | When | Then | Pass/Fail (TBD by reviewer) |
+| :---  |:---: | ---: |---:                         |
+| `precondition/initial context/state of the system. Ex: I am on Product page` | `event/trigger. Ex: I click Buy Now` |`output/action. Ex: I am redirected to Payment page` |   |
```

Comment:
could we use an example that's more specific to tech debt? I'm trying to think of how I'd fill this out. Given that I'm on the github repo, when I look at the PR list, there's no dependabot PR for updating [dependency] because it's already up to date?